### PR TITLE
Enhance Using DANDI section for new users

### PR DESCRIPTION
**Pull request by melster1010**
Added a Quick Start section up front and relocated the schematic diagram to the Developer Guide section per issue #61

### CODE REVIEW

File: docs/10_using_dandi.md [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
-## Dandisets
 
-DANDI stores cellular neurophysiology datasets. We refer to such datasets as `Dandisets`.
+If you are new to DANDI, all you need is a browser and an Internet connection to view and download part or all of a 
+public 
+`Dandiset`. 
+Registration is not required.
 
-Formally, a `Dandiset` is a collection of assets (files and their metadata) and metadata about the collection.
+## Getting Started
```
[the remark this answers is not in the document]
Comment (melster1010) [reply docs/10_using_dandi.md:9]: I totally agree. I renamed the section Quick Start vs Getting Started, to highlight the initial action someone could take right from the start, as a data consumer, which was the goal of this PR.

File: docs/10_using_dandi.md [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
+   [Download](./12_download.md) section to install and use the DANDI Python client tool.
+
+### Next steps
+See the sections that follow for more detailed information about the DANDI project, as well as instructions on how 
+to work with public `Dandisets` or to create and publish you own as a registered user.
+
+## DANDI Project Components
+
+The DANDI project contains the DANDI Web application, the DANDI Python client tool, and the DANDI JupyterHub instance. 
+DANDI stores cellular neurophysiology datasets in `Dandisets`.
```
[the remark this answers is not in the document]
Comment (melster1010) [reply docs/10_using_dandi.md:36]: I moved this section farther down, where there are already links to the components. For user doc, I'm not convinced having a figure that describes each component is as useful as info that is task-oriented.

File: docs/10_using_dandi.md [hunk 3]
```diff
@@ -1,20 +1,51 @@
 # Using DANDI
 
-## Dandisets
 
-DANDI stores cellular neurophysiology datasets. We refer to such datasets as `Dandisets`.
+If you are new to DANDI, all you need is a browser and an Internet connection to view and download part or all of a 
+public 
+`Dandiset`.
```
[the remark this answers is not in the document]
Comment (melster1010) [reply docs/10_using_dandi.md:6]: agreed, I moved the dandiset info back up

File: docs/10_using_dandi.md [hunk 4]
```diff
@@ -1,20 +1,51 @@
 # Using DANDI
 
-## Dandisets
 
-DANDI stores cellular neurophysiology datasets. We refer to such datasets as `Dandisets`.
+If you are new to DANDI, all you need is a browser and an Internet connection to view and download part or all of a 
+public
```
[the remark this answers is not in the document]
Comment (melster1010) [reply docs/10_using_dandi.md]: yes, I think identifying and fleshing out personas is a good idea, and this could be addressed in a separate issue

**Review by satra — APPROVED** [review]

**Pull request merged**